### ISSUE TITLE
Set default app for "Open in External app"

### ISSUE THREAD

**Issue by AesaraB** [mypaint/mypaint#1217]
This should a preference, and should accept either desktop entries or a path to a binary.

**Comment by odysseywestra** [comment]
Yes, so many times I try to get Krita to open a layer, but many times it just won't show up as an option.
